src/miner.h

```diff
@@ -24,6 +24,8 @@ static const int DEFAULT_GENERATE_THREADS = 1;
 
 static const bool DEFAULT_PRINTPRIORITY = false;
 
+static const bool DEFAULT_ENABLETXMININGDELAY = true;
```

Comment:
Should the value of the argument be the estimated propagation latency, rather than just a boolean value and having the latency hardcoded? That would still work with the values you've set for the flags in all the tests (setting the delay to zero) but would make the propagation latency configurable by the miner.

Reply:
I thought about that, but it is going in a different direction to the idea of automatically calculating the latency on a per-transaction basis. I think we should try the current PR out with mining pools that have been experiencing orphan rate problems first.